src/graphics/Graphviz.jl

```diff
@@ -275,4 +275,18 @@ end
 
 indent(io::IO, n::Int) = print(io, " "^n)
 
+function to_graphviz_with_viewer(g::Graph; path::String="", kw...)
```

Comment:
Thanks Matt! I would suggest calling this function `view_graphviz` since the input is already converted to a Graphviz graph.